[wrangler] dependency on libc++ (#6744)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

- `--tsconfig` {{<type>}}string{{</type>}}
  - Path to a custom `tsconfig.json` file.
- `--local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

  - Run the preview of the Worker directly on your local machine.
    {{<Aside type="warning">}}
    This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's Edge (for instance, this includes your data stored on KV). If you'd like to persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.
    {{</Aside>}}

- `--experimental-local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
  - Run the preview of the Worker directly on your local machine using the [open source Cloudflare Workers runtime](https://github.com/cloudflare/workerd).
    {{<Aside type="warning">}}
    When developing on Wrangler there there is currently a `libc++1` dependency that needs to be installed on your machine.

>     {{</Aside>}}

Author: Done.

- `--tsconfig` {{<type>}}string{{</type>}}
  - Path to a custom `tsconfig.json` file.
- `--local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

  - Run the preview of the Worker directly on your local machine.
    {{<Aside type="warning">}}

>     This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's Edge (for instance, this includes your data stored on KV). If you'd like to persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.

Reviewer: ```suggestion
    This runs an ephemeral local version of your Worker, and will not be able to access data stored on Cloudflare's network (for example, this includes your data stored on KV). To persist data locally, using the `--persist` flag will tell Wrangler to store data in the `.wrangler/state` subdirectory.
```

    {{</Aside>}}

- `--experimental-local` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
  - Run the preview of the Worker directly on your local machine using the [open source Cloudflare Workers runtime](https://github.com/cloudflare/workerd).
    {{<Aside type="warning">}}
  - When working on Wrangler, you'll need to satisfy [`workerd`](https://github.com/cloudflare/workerd)'s `libc++1` runtime dependencies:
  - On Linux:
    - libc++ (e.g. the package `libc++1` on Debian Bullseye)
  - On macOS:
    - The XCode command line tools, which can be installed with xcode-select --install
      {{</Aside>}}
- `--experimental-local-remote-kv` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}

>   - This will write/read to/from real KV namespaces. Note this flag requires `--experimental-local` to be enabled.

Reviewer 2: ```suggestion
  - This will write/read to/from your remote KV namespaces, as specified in `wrangler.toml`. Note this flag requires `--experimental-local` to be enabled.
```
@mrbbot does it use the preview namespace?

>   - When working on Wrangler, you'll need to satisfy [`workerd`](https://github.com/cloudflare/workerd)'s `libc++1` runtime dependencies:
>   - On Linux:
>     - libc++ (e.g. the package `libc++1` on Debian Bullseye)
>   - On macOS:
>     - The XCode command line tools, which can be installed with xcode-select --install
>       {{</Aside>}}
> - `--experimental-local-remote-kv` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
>   - This will write/read to/from your remote KV namespaces, as specified in `wrangler.toml`. Note this flag requires `--experimental-local` to be enabled.

Reviewer: ```suggestion

When working on Wrangler, you need to satisfy [`workerd`](https://github.com/cloudflare/workerd)'s `libc++1` runtime dependencies:

On Linux: libc++ (for example, the package `libc++1` on Debian Bullseye).
On macOS: The XCode command line tools, which can be installed with `xcode-select --install`.

      {{</Aside>}}
      
- `--experimental-local-remote-kv` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
  - This will write/read to/from your remote KV namespaces, as specified in `wrangler.toml`. Note this flag requires `--experimental-local` to be enabled.
```

Reviewer: Reformatting the aside here since it won't follow the same spacing.
